src/commitizen/adapter.js

```diff
@@ -78,6 +78,7 @@ function getNearestNodeModulesDirectory(options) {
   let nodeModulesDirectories = findNodeModules(options);
   
    // Make sure we find a node_modules folder
+  /* istanbul ignore else */
```

Comment:
Should we really add the ignore things? Potentially we could write tests for it, isn't this just tricking us into thinking we have a high code coverage?

I usually only add it when there should be 100% impossible to reach it. example:

```js
function test (a) {
  let n
  switch (a) {
    case 'one': n = 1; break
    case 'two': n = 2; break
    default: throw new Error('...')
  }

  switch (a) {
    case 'one': n += 10; break
    case 'two': n += 20; break
    /* istanbul ignore next */
    default: throw new Error('...')
  }

  return n
}
```